languages/common-lisp/exercises/concept/basics/.docs/after.md

```diff
@@ -0,0 +1,9 @@
+This exercise's goal was to begin learning about Common Lisp with a small
```

Comment:
Is this the file where we might add links to 'further reading'? Maybe we can include a link to the explanation of the 'historical reasons' for `car`/`cdr`? Good idea? Bad idea?

Reply:
I think this would be the perfect place for something like that! Do you have any particular link in mind?

Reply:
Yep, the after document is perfect for this type of information!

Reply:
I'll find a few and then we can choose the one we like the most... or perhaps just paraphrase to create our own...